router/deployment.yaml

```diff
@@ -112,6 +112,20 @@ spec:
   selector:
     app: auth
 ---
+apiVersion: v1
+kind: Service
+metadata:
+  name: workshop
+  labels:
+    app: workshop
+spec:
+  ports:
+  - port: 80
+    protocol: TCP
+    targetPort: 5000
+  selector:
+    app: notebook
```

Comment:
Should this be workshop?

Reply:
Good question.  No.  workshop is a "fake" service.  The notebook service implements both notebook and workshop, and switches off the host name when routing requests.